recipes/TextToSpeech.py

```diff
@@ -401,6 +403,18 @@ def run(self, state: dict):
                 state["audio_url"] = upload_file_from_bytes(
                     "openai_tts.mp3", response.content
                 )
+            case TextToSpeechProviders.GHANA_NLP:
+                response = requests.post(
+                    "https://translation-api.ghananlp.org/tts/v1/tts",
+                    headers=GHANA_API_AUTH_HEADERS,
+                    json={
+                        "text": text,
+                        "language": "tw",
```

Comment:
Shouldnt this use the language that the user selected

Reply:
@devxpy updated

Reply:
This should read from local var state not global gui.session state

Reply:
@devxpy updated used the state var but I noticed that open is using gui.session_state so I used the same